### PR TITLE
[SDPA-4326] fix text icon component issue

### PR DESCRIPTION
**Pull request by rubyllamadora**
**JIRA issue:** https://digital-engagement.atlassian.net/browse/SDPA-4326

### Changed

1.  Updated spacing of Icon from the text
2. Added unit test
3. Converted to stories to mdx format

Storybook: http://storybook-ripple-pr-764.lagoon.vicsdp.amazee.io/?path=/docs/atoms-icon--text-icon
### Screenshots

image

### CODE REVIEW

File: packages/components/Atoms/Icon/TextIcon.vue [hunk 1]
```diff
@@ -15,6 +15,10 @@
 <script>
 import RplIcon from './Icon.vue'
 
+/**
+ * Text Icon allows user to display text with an icon that can be placed before or after.
+ * This is commonly used in an anchor <a> tag where the icon helps represents the action of the text.
+ */
```
Comment (rubyllamadora) [inline packages/components/Atoms/Icon/TextIcon.vue:21]: I feel like component description or component in general should be owned by UX/UI? Any suggestions to description?

Comment (tim-yao) [reply packages/components/Atoms/Icon/TextIcon.vue:21]: @rubyllamadora Yes, I will copy them from Ripple design file, you can find them in Zeplin.

**Review by tim-yao — APPROVED** [review]
Thanks @rubyllamadora It's nice to see all the unit tests and docs got added in this ticket!

File: packages/components/Atoms/Icon/TextIcon.vue (hunk 1 above)
Comment (rubyllamadora) [reply packages/components/Atoms/Icon/TextIcon.vue:21]: Thanks @tim-yao. Will look for them in zeplin and will update once I find the description.

**Pull request merged**